scripts/visualize.py

```diff
@@ -36,6 +39,20 @@
 flags.mark_flag_as_required("dataset_version")
 
 
+def prepare_example(example):
```

Comment:
At the moment this is for the machine learning inference. More "user" (ML engineer) friendly to send versions of dataset. Otherwise I would have to send the object, the wind_speed and other conditions manually. Also, I need the ground truth, which is already stored in the dataset.

Later, when serving the model we would have to send the object, yes